src/modules/land_detector/FixedwingLandDetector.h

```diff
@@ -57,48 +57,46 @@ class FixedwingLandDetector : public LandDetector
 	FixedwingLandDetector();
 
 protected:
-	virtual void _initialize_topics() override;
+	void _initialize_topics() override;
+	void _update_params() override;
+	void _update_topics() override;
+	bool _get_landed_state() override;
+	bool _get_maybe_landed_state() override;
+	bool _get_ground_contact_state() override;
+	bool _get_freefall_state() override;
+	float _get_max_altitude() override;
 
-	virtual void _update_params() override;
-
-	virtual void _update_topics() override;
-
-	virtual bool _get_landed_state() override;
-
-	virtual bool _get_maybe_landed_state() override;
-
-	virtual bool _get_ground_contact_state() override;
```

Comment:
nitpicky: since you take out the virtual keyword, i assume you are certain that that there are no other class that will inherit from fixedwing. If that is the case, you could also use "final" keyword to indicate that it cannot be derived from fixedwing.